FF1Blazorizer/wwwroot/tooltips/tooltips.json

```diff
@@ -1,1881 +1,1905 @@
+	},
+
+	{
+		"Id": "EnableRandomPromotions",
+		"title": "Random Promotions",
+		"screenshot": "EnableRandomPromotions.gif",
+		"description": "When giving the TAIL to Bahamut, the wise dragon will promote your party to various, random classes. Random classes are pre-selected by the randomizer when the seed is generated. You will not know ahead of time what that promoted class will be."
+	},
+	{
+		"Id": "RPIncludeBaseClasses",
+		"title": "Include Base Classes",
+		"screenshot": "RPIncludeBaseClasses.gif",
+		"description": "Base classes can now be in the possible promotion pool when returning the TAIL to Bahamut."
+	},
+	{
+		"Id": "RPSpoilers",
+		"title": "Spoil Promotions",
+		"screenshot": "RPSpoilers.png",
+		"description": "The names of the various classes with be changed to show what classes they will become after promotions.\nFormat: Current Class - Promotion Class"
+	},
+
+	{
+		"Id": "ChangeMaxMP",
+		"title": "Change Spellcaster Max MP",
+		"screenshot": "ChangeMaxMP.png",
+		"description": "Allows control over the maximum spell charges per spell level for each spellcasting class in the game. This does not change when they receive new spell levels (as that's tied to experience level). A value of less than 2 will not reduce the starting Level 1 charges of Red, white, and Black mages."
+	},
+	{
+		"Id": "AllSpellLevelsForKnightNinja",
+		"title": "Knight and Ninja Gain Charges in All Levels",
+		"screenshot": "AllSpellLevelsForKnightNinja.png",
+		"description": "Sets the Knight and Ninja magic charge growth to give a charge in all levels whenever those classes gain a charge in any level, giving Knight and Ninja tier 4-8 charges. This alleviates an issue with \"Keep Permissions\" where the Knight and Ninja could never cast spells they could learn, but is useless otherwise."
+	},
+
+
+
+	{
+		"Id": "startingEquipmentHealStaff",
+		"title": "Starting Equipment - Heal Staff",
+		"screenshot": "startingEquipmentHealStaff.png",
+		"description": "Places a Heal Staff (or any low level Heal casting weapon, or a Heal Helment if nothing else is found in the game) into the party's starting inventory."
+	},
+	{
+		"Id": "startingEquipmentDragonslayer",
+		"title": "Starting Equipment - Dragonslayer",
+		"screenshot": "startingEquipmentDragonslayer.png",
+		"description": "Places the Dragon Armor and the Dragon Sword into the party's starting inventory.\nIncompatible with Weaponizer."
+	},
+	{
+		"Id": "startingEquipmentLegendKit",
+		"title": "Starting Equipment - Legendary Kit",
+		"screenshot": "startingEquipmentLegendKit.png",
+		"description": "Playes the Xcalibur, Opal Armor, Opal Gauntlets, Opal Helm, and Opal Shield into the party's starting inventory.\nIncompatible with Weaponizer."

```

Comment:
Its referred as Xcalber somewhere else in this file. I'm not sure what's the correct name.

Reply:
Xcalber is the right used by the game;
also Playes > Places